.github/pull_request_template.md

```diff
@@ -1,30 +1,47 @@
-(Thanks for sending a pull request! Please fill out the blanks below.)
+<!--
+This Pull Request Template is a modified version from Vuejs's:
+https://raw.githubusercontent.com/vuejs/vue/dev/.github/PULL_REQUEST_TEMPLATE.md
+-->
 
-What does this implement/fix? Explain your changes
----------------------------------------------------
-...
+<!-- PULL REQUEST TEMPLATE -->
+<!-- (Update "[ ]" to "[x]" to check a box) -->
 
-Does this close any currently open issues?
-------------------------------------------
-(Use the special github keywords to reference issues: <https://help.github.com/en/articles/closing-issues-using-keywords>)
+**Describe what this PR introduces:**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=5218633)